behave/features/1.user_journey_happy_path.feature

```diff
@@ -0,0 +1,114 @@
+# COVID-19 - Happy-path Feature
+@happy_path
```

Comment:
rename this file to say it does not use nhs login

Reply:
The file has now been renamed as requested